docs/sphinx/source/user_guide/nomenclature.rst

```diff
@@ -142,6 +142,12 @@ There is a convention on consistent variable names throughout the library:
     solar_zenith
         Zenith angle of the sun in degrees
 
+    spectra, spectra_components
+        Spectral irradiance (components) [Wm⁻²nm⁻¹]. The amount of energy
+        incident on a horizontal unit area per unit time and per unit
```

Comment:
```suggestion
        incident on a unit area per unit time and per unit
```

No need for it to be horizontal (e.g. `spectrl2` gives tilted spectral irradiance).